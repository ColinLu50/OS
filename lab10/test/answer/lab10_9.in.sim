16123
729
691
473
687
470
