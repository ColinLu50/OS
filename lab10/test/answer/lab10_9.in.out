FCFS
217 -> 53 -> 276 -> 330 -> 432 -> 385 -> 314 -> 474 -> 322 -> 240 -> 471 -> 377 -> 224 -> 292 -> 68 -> 392 -> 174 -> 3 -> 404 -> 37 -> 357 -> 320 -> 437 -> 408 -> 141 -> 441 -> 325 -> 190 -> 10 -> 407 -> 149 -> 322 -> 283 -> 455 -> 298 -> 464 -> 312 -> 332 -> 47 -> 41 -> 425 -> 140 -> 53 -> 453 -> 266 -> 56 -> 240 -> 242 -> 364 -> 80 -> 425 -> 294 -> 231 -> 457 -> 341 -> 191 -> 227 -> 174 -> 125 -> 88 -> 177 -> 397 -> 474 -> 127 -> 380 -> 135 -> 257 -> 358 -> 175 -> 473 -> 41 -> 248 -> 366 -> 348 -> 220 -> 304 -> 422 -> 37 -> 2 -> 424 -> 468 -> 29 -> 27 -> 145 -> 209 -> 242 -> 110 -> 393 -> 282 -> 401 -> 147 -> 79 -> 219 -> 384 -> 98
16123
SSTF
217 -> 219 -> 220 -> 224 -> 227 -> 231 -> 240 -> 240 -> 242 -> 242 -> 248 -> 257 -> 266 -> 276 -> 282 -> 283 -> 292 -> 294 -> 298 -> 304 -> 312 -> 314 -> 320 -> 322 -> 322 -> 325 -> 330 -> 332 -> 341 -> 348 -> 357 -> 358 -> 364 -> 366 -> 377 -> 380 -> 384 -> 385 -> 392 -> 393 -> 397 -> 401 -> 404 -> 407 -> 408 -> 422 -> 424 -> 425 -> 425 -> 432 -> 437 -> 441 -> 453 -> 455 -> 457 -> 464 -> 468 -> 471 -> 473 -> 474 -> 474 -> 209 -> 191 -> 190 -> 177 -> 175 -> 174 -> 174 -> 149 -> 147 -> 145 -> 141 -> 140 -> 135 -> 127 -> 125 -> 110 -> 98 -> 88 -> 80 -> 79 -> 68 -> 56 -> 53 -> 53 -> 47 -> 41 -> 41 -> 37 -> 37 -> 29 -> 27 -> 10 -> 3 -> 2
729
SCAN
217 -> 209 -> 191 -> 190 -> 177 -> 175 -> 174 -> 149 -> 147 -> 145 -> 141 -> 140 -> 135 -> 127 -> 125 -> 110 -> 98 -> 88 -> 80 -> 79 -> 68 -> 56 -> 53 -> 47 -> 41 -> 37 -> 29 -> 27 -> 10 -> 3 -> 2 -> 0 -> 219 -> 220 -> 224 -> 227 -> 231 -> 240 -> 242 -> 248 -> 257 -> 266 -> 276 -> 282 -> 283 -> 292 -> 294 -> 298 -> 304 -> 312 -> 314 -> 320 -> 322 -> 325 -> 330 -> 332 -> 341 -> 348 -> 357 -> 358 -> 364 -> 366 -> 377 -> 380 -> 384 -> 385 -> 392 -> 393 -> 397 -> 401 -> 404 -> 407 -> 408 -> 422 -> 424 -> 425 -> 432 -> 437 -> 441 -> 453 -> 455 -> 457 -> 464 -> 468 -> 471 -> 473 -> 474
691
C-SCAN
217 -> 209 -> 191 -> 190 -> 177 -> 175 -> 174 -> 149 -> 147 -> 145 -> 141 -> 140 -> 135 -> 127 -> 125 -> 110 -> 98 -> 88 -> 80 -> 79 -> 68 -> 56 -> 53 -> 47 -> 41 -> 37 -> 29 -> 27 -> 10 -> 3 -> 2 -> 0 -> 475 -> 474 -> 473 -> 471 -> 468 -> 464 -> 457 -> 455 -> 453 -> 441 -> 437 -> 432 -> 425 -> 424 -> 422 -> 408 -> 407 -> 404 -> 401 -> 397 -> 393 -> 392 -> 385 -> 384 -> 380 -> 377 -> 366 -> 364 -> 358 -> 357 -> 348 -> 341 -> 332 -> 330 -> 325 -> 322 -> 320 -> 314 -> 312 -> 304 -> 298 -> 294 -> 292 -> 283 -> 282 -> 276 -> 266 -> 257 -> 248 -> 242 -> 240 -> 231 -> 227 -> 224 -> 220 -> 219
473
LOOK
217 -> 209 -> 191 -> 190 -> 177 -> 175 -> 174 -> 149 -> 147 -> 145 -> 141 -> 140 -> 135 -> 127 -> 125 -> 110 -> 98 -> 88 -> 80 -> 79 -> 68 -> 56 -> 53 -> 47 -> 41 -> 37 -> 29 -> 27 -> 10 -> 3 -> 2 -> 219 -> 220 -> 224 -> 227 -> 231 -> 240 -> 242 -> 248 -> 257 -> 266 -> 276 -> 282 -> 283 -> 292 -> 294 -> 298 -> 304 -> 312 -> 314 -> 320 -> 322 -> 325 -> 330 -> 332 -> 341 -> 348 -> 357 -> 358 -> 364 -> 366 -> 377 -> 380 -> 384 -> 385 -> 392 -> 393 -> 397 -> 401 -> 404 -> 407 -> 408 -> 422 -> 424 -> 425 -> 432 -> 437 -> 441 -> 453 -> 455 -> 457 -> 464 -> 468 -> 471 -> 473 -> 474
687
C-LOOK
217 -> 209 -> 191 -> 190 -> 177 -> 175 -> 174 -> 149 -> 147 -> 145 -> 141 -> 140 -> 135 -> 127 -> 125 -> 110 -> 98 -> 88 -> 80 -> 79 -> 68 -> 56 -> 53 -> 47 -> 41 -> 37 -> 29 -> 27 -> 10 -> 3 -> 2 -> 474 -> 473 -> 471 -> 468 -> 464 -> 457 -> 455 -> 453 -> 441 -> 437 -> 432 -> 425 -> 424 -> 422 -> 408 -> 407 -> 404 -> 401 -> 397 -> 393 -> 392 -> 385 -> 384 -> 380 -> 377 -> 366 -> 364 -> 358 -> 357 -> 348 -> 341 -> 332 -> 330 -> 325 -> 322 -> 320 -> 314 -> 312 -> 304 -> 298 -> 294 -> 292 -> 283 -> 282 -> 276 -> 266 -> 257 -> 248 -> 242 -> 240 -> 231 -> 227 -> 224 -> 220 -> 219
470
